main.go

```diff
@@ -24,6 +26,23 @@ func init() {
 	flag.StringVar(&cacheDir, "cacheDir", "", "go modules cache dir")
 	flag.StringVar(&listen, "listen", "0.0.0.0:8081", "service listen address")
 	flag.Parse()
+	isGitValid := checkGitVersion()
+	if !isGitValid {
+		log.Fatal("Error in git version, please check your git installed in local, must be great 2.0")
+	}
+}
+
+func checkGitVersion() bool {
+	var err error
+	var ret []byte
+	cmd := exec.Command("/usr/bin/git", "version")
```

Comment:
The last question: `/usr/bin/git` seems not right. This just a default path for git.

Reply:
fix it, changes it to "git version", use the local environment.